spec/page_element_spec.rb

```diff
@@ -22,20 +22,41 @@
       )).pages.first
   end
 
-  context '#title' do
-    context 'no metadata' do
+  let(:page2) do
+    book_containing(html:
+      <<~HTML
+        <div data-type="page">
+          <div data-type="document-title">Title</div>
+          <div data-type="metadata">
+            <div data-type="document-title">Title MetaData</div>
+          </div>
+        </div>
+      HTML
+    )
+  end
+
+  describe '#title' do
+    context 'with no metadata' do
       let(:metadata) { '' }
 
       it 'finds the title' do
-        expect(page_1.title.text).to eq page_title_text
+        expect(page1.title.text).to eq page_title_text
       end
     end
 
     context 'with metadata' do
       it 'finds the title' do
-        expect(page_1.title.text).to eq page_title_text
+        expect(page1.title.text).to eq page_title_text
       end
     end
   end
 
+  describe '#titles' do
+    it 'returns all titles in element' do
+      page_titles = '<div data-type="document-title">Title</div>'\
+      '<div data-type="document-title">Title MetaData</div>'
+      expect(page2.pages[0].titles.to_s).to eq page_titles
```

Comment:
See comment about above a different variable name and then you can simplify these three lines to

```ruby
expect(page_with_multiple_titles.titles.to_s).to match(/Title.*Title Metadata/)
```

or 

```ruby
expect(page_with_multiple_titles.titles.map(&:to_s)).to match([/Title/, /Title MetaData/])
```

using a regular expression(s) to match the parts you care about